### PR TITLE
RELATED: RAIL-1815 - Sweeping in sdk-ui-charts/highcharts

### PR DESCRIPTION
**Pull request by lupko**
Move refactors and restructuralizations. Preparation for further refactors

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-ui-charts/src/highcharts/index.ts [hunk 1]
```diff
@@ -27,10 +27,10 @@ export {
     isPieChart,
     isPieOrDonutChart,
     isBulletChart,
-} from "./utils/common";
+} from "./chartTypes/_util/common";
 
 export { BOTTOM, MIDDLE, TOP } from "./constants/alignments";
```
Comment (no23reason) [inline libs/sdk-ui-charts/src/highcharts/index.ts:32]: Should this too come from the `mess.ts` file?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1297/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2891/ : SUCCESS in 13m 33s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/462/ : SUCCESS in 29s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/721/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2893/ : SUCCESS in 13m 26s

**Pull request merged**